dist/obs-server.spec

```diff
@@ -92,13 +92,7 @@ Requires:       perl(Date::Parse)
 Requires:       diffutils
 PreReq:         git-core
 Requires:       patch
-# require the createrepo and python-yum version which got validated during testsuite run
-%if 0%{?suse_version} < 1500
-Requires:       %(/bin/bash -c 'rpm -q --qf "%%{name} = %%{version}-%%{release}" createrepo')
-Requires:       %(/bin/bash -c 'rpm -q --qf "%%{name} = %%{version}-%%{release}" python-yum')
-%else
-Requires:       /usr/bin/createrepo
-%endif
+Requires:       createrepo_c
```

Comment:
why would you require createrep_c if all testing in package builds is done with createrepo?